Failed to activate the markdown-themeable-pdf package

[Enter steps to reproduce:]

1. I got the error: "ENOENT: no such file or directory, scandir '/Users/steevechwojko/.atom/packages/markdown-themeable-pdf-1.2.0/node_modules/highlight.js/styles'

2. I manually created this path to see whether it would make a difference. The new error was as shown below. 

**Atom**: 1.14.4 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.3
**Thrown From**: [markdown-themeable-pdf](https://github.com/cakebake/markdown-themeable-pdf) package 1.2.0


### Stack Trace

Failed to activate the markdown-themeable-pdf package

```
At Cannot find module 'ncp'

Error: Cannot find module 'ncp'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._resolveFilename (/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (/app.asar/src/native-compile-cache.js:50:27)
    at Object.activate (/packages/markdown-themeable-pdf-1.2.0/lib/markdown-themeable-pdf.js:211:33)
    at Package.module.exports.Package.activateNow (/app.asar/src/package.js:218:19)
    at /app.asar/src/package.js:190:32
    at Package.module.exports.Package.measure (/app.asar/src/package.js:96:15)
    at /app.asar/src/package.js:183:26
    at Package.module.exports.Package.activate (/app.asar/src/package.js:180:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/app.asar/src/package-manager.js:550:34)
    at /app.asar/src/package-manager.js:531:29
    at Config.module.exports.Config.transactAsync (/app.asar/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/app.asar/src/package-manager.js:526:19)
    at PackageManager.module.exports.PackageManager.activate (/app.asar/src/package-manager.js:508:46)
    at /app.asar/src/atom-environment.js:814:28
```

### Commands

```
```

### Non-Core Packages

```
markdown-pdf 1.5.0 
markdown-themeable-pdf 1.2.0 
```

--- a/lib/markdown-themeable-pdf.js
+++ b/lib/markdown-themeable-pdf.js
@@ -209,7 +209,7 @@ module.exports = markdownThemeablePdf = {
         );
 
         var ncp = require('ncp').ncp;
-        var customTemplatesPath = path.resolve(atom.config.configDirPath, 'markdown-themeable-pdf');
+        var customTemplatesPath = path.resolve(atom.getConfigDirPath(), 'markdown-themeable-pdf');
         var customTemplatesSource = path.resolve(__dirname, '../markdown-themeable-pdf');
         ncp(customTemplatesSource, customTemplatesPath, {clobber: false}, function (err) {
             if (!err)

